src/org/scada_lts/dao/model/SimpleScadaEntity.java

```diff
@@ -0,0 +1,40 @@
+package org.scada_lts.dao.model;
+
+public class SimpleScadaEntity {
```

Comment:
I've replaced simple `JsonMailingList` class with `SimpleScadaEntity` class. It has the same properties like id, xid and name, but now the name is not so strictly bound to any specific Scada object. Now it is more general class that can be used to display list of Scada-LTS business objects like MailingLists, WatchLists, Users, Event Handlers and so on.

Reply:
The point is to include in the name what data it contains, I would avoid the word entity here, because this structure has no representation in the database, it is more logical.
maybe: ScadaIdentifier, PrimaryIdentifier, BasicIdentifier